modules/account-lib/package.json

```diff
@@ -52,6 +52,7 @@
     "elliptic": "^6.5.2",
     "ethereumjs-abi": "^0.6.5",
     "ethereumjs-util": "5.2.0",
+    "ethereumjs-utils-old": "npm:ethereumjs-utils@5.2.0",
```

Comment:
why is one package `ethereumjs-utils` (plural) and the original one is `ethereumjs-util` (singular)?

Seems to me like these are totally separate packages:

* https://www.npmjs.com/package/ethereumjs-util
* https://www.npmjs.com/package/ethereumjs-utils

Reply:
Nice catch. I will update